lms/static/js/student_account/components/UnsubscribeMarketingEmail.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CheckBox, CheckBoxGroup } from '@edx/paragon/static';
+import { unsubscribe, subscribe } from "../AccountsClient";
+
+
+
+class UnsubscribeMarketingEmail extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: props.isUserSubscribed,
+    };
+    this.subscribeMarketingEmail = this.subscribeMarketingEmail.bind(this);
+    this.unsubscribeMarketingEmail = this.unsubscribeMarketingEmail.bind(this);
+    this.handleCheckbox = this.handleCheckbox.bind(this);
+  }
+
+  handleCheckbox () {
+    this.state.checked ? this.unsubscribeMarketingEmail(): this.subscribeMarketingEmail()
+  }
+
+  subscribeMarketingEmail() {
+    subscribe(this.props.userEmail)
+        .then(() => this.setState({
+          checked: true
+        }))
+        .catch(error => console.log(error.message))
+  }
+
+  unsubscribeMarketingEmail() {
+    unsubscribe(this.props.userEmail)
+        .then(() => this.setState({
+          checked: false
+        }))
+        .catch(error => console.log(error.message))
+  }
+
+  render() {
+    const { checked } = this.state;
+
+    return (
+      <div className="account-deletion-details">
+        <CheckBoxGroup>
+          <CheckBox onChange={this.handleCheckbox} checked={checked} />
+          <p className='subscription-text'>
+            {gettext('Subscribe to marketing email')}
+          </p>
+          <p className='subscription-text'>
+            {gettext('Please note: Once your account is unsubscribed from marketing email, you will no longer receive email from edraak.org, or any other site hosted by Edraak.')}
+          </p>
+        </CheckBoxGroup>
```

Comment:
I believe we should use (emails) not (email). Maybe technically it is a subscription to a (Marketing Email), but I see it commonly used as plural (Marketing Emails) everywhere. Users see it as email **messages**, not email addresses
**_Note:_** you'll have to re-translate if you apply this amendment

On the other hand, please provide the required props for paragon `CheckBox` to avoid browser logs. And I don't see it necessary to use the `CheckBoxGroup`
```suggestion
        <CheckBox
          onChange={this.handleCheckbox}
          checked={checked}
          name='unsubscribe_checkbox'
          label=''
        />
        <p className='subscription-text'>
          {gettext('Subscribe to marketing emails')}
        </p>
        <p className='subscription-text'>
          {gettext('Please note: Once your account is unsubscribed from marketing emails, you will no longer receive emails from edraak.org, or any other site hosted by Edraak.')}
        </p>
```